packages/experience/src/hooks/use-start-backup-code-binding.ts

```diff
@@ -0,0 +1,47 @@
+import { MfaFactor, VerificationType } from '@logto/schemas';
+import { useCallback, useContext } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import UserInteractionContext from '@/Providers/UserInteractionContextProvider/UserInteractionContext';
+import { createBackupCode } from '@/apis/experience';
+import { UserMfaFlow } from '@/types';
+import { type BackupCodeBindingState } from '@/types/guard';
+
+import useApi from './use-api';
+import useErrorHandler from './use-error-handler';
+
+type Options = {
+  /** Whether to replace the current page in the history stack. */
```

Comment:
still confusing.

this hook is returning a function, and this option is affecting the behavior in that function - should specify this or remove this param

Reply:
Update all legacy hooks, remove this param from hooks. Should pass it in the callback method directly.